policy/modules/system/init.if

```diff
@@ -1451,6 +1451,27 @@ interface(`init_relabel_var_lib_dirs',`
 	allow $1 init_var_lib_t:dir relabel_dir_perms;
 ')
 
+########################################
+## <summary>
+##	Create, read, write, and delete the
+##	pseudorandom number generator seed.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`init_manage_random_seed',`
+	gen_require(`
+		type init_random_seed_t;
+	')
+
+	files_search_var_lib($1)
+	files_search_runtime($1)
+	manage_files_pattern($1, init_random_seed_t, init_random_seed_t)
```

Comment:
Considering the labeling in the `.fc`, this seems excessive since there is no `init_random_seed_t` dirs.

Reply:
Good point, I have just amended it as suggested ! Thanks.